docs/DeveloperGuide.md

```diff
@@ -154,90 +155,44 @@ Classes used by multiple components are in the `seedu.address.commons` package.
 
 This section describes some noteworthy details on how certain features are implemented.
 
-### \[Proposed\] Undo/redo feature
+Usage of `editClient` and `editMeeting`:
 
-#### Proposed Implementation
+`editClient` and `editMeeting` execute in a similar manner to each other.
+Let the term entity refer to either a client or a meeting.
+Every entity is uniquely identified by a UUID in storage, so essentially all fields in an entity can be edited without loss of uniqueness of the entity.
```

Comment:
Is this all you have for editClient and editMeeting? i think you should talk about the implementation of it than give other alternative and why other alternatives wasn't chosen.

Reply:
okay, I'll work on it